src/main/docker/app.yml

```diff
@@ -1,35 +1,25 @@
 version: '2'
 services:
-    resource-app:
```

Comment:
@yevheniimuzyka 
why TAB width was changed from 4 to 2 spaces? it produces a lot of changes.